notebooks/001-getting-started/001-getting-started.ipynb

```diff
@@ -54,10 +54,13 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.    logger = logging.getLogger("anomalib")
</pre><p>Do we need to import and define the logger here? It doesn't seem to be used in any of the cells. Maybe check the other imports as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/357/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>